e2e/utils/TestUtils.ts

```diff
@@ -37,13 +37,52 @@ export function playwrightReportAssets(fileName: string) {
   return path.join(__dirname, '..', 'reports', `${ fileName }-pw-trace.zip`);
 }
 
+/**
+ * Custom helm home and cache directories
+ * Matt tip that will help to remove all helm test data from the SUT
+ */
+export function setUpHelmCustomEnv() {
+  const tempHelmFolder = path.join(os.homedir(), 'helmTmp');
+  const helmRepoCacheFolder = path.join(tempHelmFolder, 'Caches', 'helm', 'repository');
+  const helmRepoConfigFolder = path.join(tempHelmFolder, 'Config', 'helm', 'repository');
+
+  process.env.HELM_REPOSITORY_CACHE = helmRepoCacheFolder;
+  process.env.HELM_REPOSITORY_CONFIG = `${ helmRepoConfigFolder }/repositories.yaml`;
+
+  if (!fs.existsSync(tempHelmFolder)) {
+    mkdirSync(tempHelmFolder, { recursive: true });
+  }
+}
+
 /**
  * helm teardown
  * it ensure that all helm test installation contents will be deleted.
  */
-export async function tearDownHelm() {
-  await helm('repo', 'remove', 'bitnami');
-  await kubectl('delete', 'deploy', 'nginx-sample', '--namespace', 'default');
+export function tearDownHelm() {
+  const helmTempPath = path.join(os.homedir(), 'helmTmp');
+
+  if (fs.existsSync(helmTempPath)) {
+    fs.rmSync(helmTempPath, { recursive: true });
+  }
+}
+
+/**
+ * Detects which platform the spec is running and returns
+ * the platform name string.
```

Comment:
This seems unnecessary. Just check for an unrecognized platform through a hash object where `detectPlatform` is invoked.